sql-parser/src/test/java/io/crate/sql/parser/TestStatementBuilder.java

```diff
@@ -142,6 +142,12 @@ public void testStatementBuilder()
         printStatement("create table \"TABLE\" (o object(ignored))");
         printStatement("create table \"TABLE\" (o object(strict) as (inner_col object as (sub_inner_col timestamp, another_inner_col string)))");
 
+        printStatement("create table test (col1 int, col2 timestamp not null) partitioned by (col1)");
```

Comment:
pls revise tests here if they are all needed, specially the partitioned cause can be skipped here because afaik not relevant. I think it can be stripped to just 2 cases, one for single not null contraint and another one for multiple constraints...